kimchi/src/error.rs

```diff
@@ -20,6 +20,9 @@ pub enum ProverError {
 
     #[error("the lookup failed to find a match in the table")]
     ValueNotInTable,
+
+    #[error("kimchi does not support SRS of size smaller than the domain")]
```

Comment:
Nit: remove `kimchi` from the message? Something like `SRS size is smaller than the domain size required by the circuit`?